plugins/by-name/smartcolumn/default.nix

```diff
@@ -0,0 +1,80 @@
+{
+  lib,
+  ...
+}:
+let
+  inherit (lib.nixvim) defaultNullOpts;
+  inherit (lib) types;
+in
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "smartcolumn";
+  originalName = "smartcolumn.nvim";
+  package = "smartcolumn-nvim";
+
+  maintainers = [ lib.maintainers.khaneliman ];
+
+  settingsOptions = {
+    colorcolumn = defaultNullOpts.mkNullable (with types; either str (listOf str)) "80" ''
+      Column with to highlight.
+      Supports multiple values for more column highlights.
+    '';
```

Comment:
Seems strange upstream would expect a string :thinking:

Reply:
But it is correct https://github.com/m4xshen/smartcolumn.nvim?tab=readme-ov-file#-configuration